src/EPR.Payment.Facade/EPR.Payment.Facade.csproj.user

```diff
@@ -2,7 +2,7 @@
 <Project ToolsVersion="Current" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
This file should not be checked in. Please remove it from the solution and add it to gitignore

Reply:
Removed file and added to gitignore